core/conversion/converters/impl/interpolate.cpp

```diff
@@ -11,12 +14,75 @@ namespace converters {
 namespace impl {
 namespace {
 
+/*
+ * Helper functions
+ */
+
+auto parse_nearest(args args) {

```

Comment:
Honestly these parse functions seem kind of redundant considering they don't reduce the amount of code and you still need to do tuple unpacking.

Reply:
I can do a one-line tuple unpacking, but this will only work on C++17. Should I do this, or remove the parsing entirely?

Reply:
I think it is better to just remove it for now